acceptance/lib/puppet/acceptance/mount_utils.rb

```diff
@@ -0,0 +1,56 @@
+module Puppet
+  module Acceptance
+    module MountUtils
+
+      # Return the absolute path to the filesystem table file.
+      # @param host [String] hostname
+      # @return [String] path to the filesystem table file.
+      def filesystem_file(host)
+        if host['platform'] =~ /aix/
+          '/etc/filesystems'
+        else
+          '/etc/fstab'
```

Comment:
since we are pulling this out into a utility helper, we should guard more stringently for other platforms. For example, it looks like solaris uses `/etc/vfstab`. The default might need to be an error case.

Serving suggestion:
        case host['platform']
        when /aix/
          '/etc/filesystems'
        when /solaris/
          '/etc/vfstab'
        when /el||fedora|sles|debian|ubuntu/
          '/etc/fstab'
        else
          ''
        end